app/src/main/java/org/eyeseetea/malariacare/domain/entity/Question.java

```diff
@@ -1,25 +1,36 @@
 package org.eyeseetea.malariacare.domain.entity;
 
+import android.support.annotation.NonNull;
+
+import static org.eyeseetea.malariacare.domain.utils.RequiredChecker.required;
+
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
-import static org.eyeseetea.malariacare.domain.utils.RequiredChecker.required;
+import java.util.Map;
 
 public class Question {
```

Comment:
I still do not understand the logic that exists in question, can you create tests for questions that validate all this added logic that serves as documentation?